glossary/SQL-injection.md

```diff
@@ -0,0 +1,3 @@
+### SQL injection
+
+SQL injection is a code injection technique, used to attack data-driven applications.
```

Comment:
Maybe add a second line:

`SQL injections get their name from the SQL language and mainly target data stored in relational databases.`